app/renderer/components/tabs/content/closeTabIcon.js

```diff
@@ -17,8 +17,7 @@ const frameStateUtil = require('../../../../../js/state/frameStateUtil')
 
 // Styles
 const {theme} = require('../../styles/theme')
-const {spacing, zindex} = require('../../styles/global')
-const {opacityIncreaseKeyframes} = require('../../styles/animations')
+const globalStyles = require('../../styles/global')
```

Comment:
Before doing that task, let's open an issue to edit [this section on docs/style.md](https://github.com/brave/browser-laptop/blob/master/docs/style.md#where-to-put-my-code) to achieve a common understanding.